sentry_sdk/integrations/django/caching.py

```diff
@@ -0,0 +1,101 @@
+import functools
+from typing import TYPE_CHECKING
+
+from django import VERSION as DJANGO_VERSION
+from django.core.cache import CacheHandler
+
+from sentry_sdk import Hub
+from sentry_sdk.consts import OP, SPANDATA
+from sentry_sdk._compat import text_type
+
+
+if TYPE_CHECKING:
+    from typing import Any
+    from typing import Callable
+
+
+METHODS_TO_INSTRUMENT = [
```

Comment:
do we want to patch any others here? no setters right?

Reply:
currently it is just the getters, because we want to record cache hit or miss.